### DISCUSSION TITLE
Generate new feeds based on your filters

### DISCUSSION THREAD

**Discussion by cedriclocqueneux** [FreshRSS/FreshRSS#6549] [Q&A] [answered]
Hello. In the description of FreshRSS, I see “Generate new feeds based on your filters.”. That's exactly what I'm looking for! Tag articles with certain words, and get an rss feed on those words with all the tagged articles.
But how do I get the link to the generated rss feed? I've scoured the documentation, but I can't find the info.

**Comment by Alkarex** [comment, marked as answer]
Hello,
See https://freshrss.github.io/FreshRSS/en/users/user_queries.html#share-your-user-queries

**Reply by cedriclocqueneux** [reply]
Thank you very much !
